newton/_src/utils/import_mjcf.py

```
@@ -841,6 +850,64 @@ def parse_shapes(
+                from ..geometry.inertia import (  # noqa: PLC0415
+                    compute_inertia_box_from_mass,
+                    compute_inertia_capsule,
+                    compute_inertia_cylinder,
+                    compute_inertia_ellipsoid,
+                    compute_inertia_sphere,
+                )
+
+                # Compute inertia by calling functions with density=1.0, then scale by mass ratio
+                # This avoids manual volume computation - functions handle it internally
+                c = wp.vec3()  # center of mass (at origin for primitives)
+                I = wp.mat33()
```

Ruff (0.15.2)

[error] Ambiguous variable name: `I`

(E741)